components/mailbox/src/Mailbox.svelte

```diff
@@ -650,6 +650,16 @@
       gap: 8px;
     }
 
+    &.loading {
+      grid-template-rows: minmax(auto, calc(100vh - 16px));
+    }
+    &.empty {
+      grid-template-rows: 68px 33.5px minmax(auto, calc(100vh - 117.5px));
```

Comment:
would love to hear suggestions as to how to avoid magic numbers here

Reply:
I don't think there's much that can be done for magic numbers in css. Assign them to variables if you're going to reuse them elsewhere, but otherwise maybe just add some comments about where the numbers are coming from